### ISSUE TITLE
Consider adding `@bigtest/interactor` to package.json on `bigtest init`

### ISSUE THREAD

**Issue by cowboyd** [bigtestjs/cli#26]
As part of writing the guides, @SamKeathley put in a step to install the `@bigtest/interactor` package. This seems like a slam dunk for something that we should be doing automatically. I mean, if yiou aren't using interactor, then what the hell are you doing?

https://github.com/bigtestjs/bigtestjs.io/blob/81442cce44dba5fc454db42f32b2d594a77d21c5/source/guides/getting-started/quick-start.html.md

### Questions

Should it use yarn? Maybe if it detects a yarn executable in the path and a`yarn.lock`. Otherwise, maybe we can have a `--yarn` option to use yarn even in the event that there was not a `yarn.lock`. Although a `yarn` executable would need to be in the path.

**Comment by Robdel12** [comment]
Yeah we'll totally do this. And if there's both in the project prompt for npm or yarn.
